### DISCUSSION TITLE
Picker per Symbol type (func, var, type, struct, prop, ...)

### DISCUSSION THREAD

**Discussion by edukey** [helix-editor/helix#9944] [Ideas] [answered]
As a vscode user willing to switch to helix, I search for a feature allowing to pick for symbols only of a specific type (function, property, glob var, loc var, type, ..).

This picker is mainly for current file, but usefull also in full workspace.

This will allow similar feature as the "outline" from IDEs.

See also #7228

**Comment by the-mikedavis** [comment, marked as answer]
See https://github.com/helix-editor/helix/issues/5446 and https://github.com/helix-editor/helix/pull/9647

We plan to add a way to filter the symbol pickers by symbol kind rather than introducing pickers per symbol kind

**Reply by edukey** [reply]
Great, sorry I missed PR #7265 from #9647 

note: having to type `%kind:func` seems a bit heavy when regulary searching for functions
hoping it will be possible to have shortcuts like `%k:f` or `%kf` or `%f` 
my ideal will be a prefix as `f:`  , ex: `f:deser` to search for function symbols containing `deser`

**Reply by the-mikedavis** [reply]
The latest from #9647 adds that prefix syntax and the values are fuzzy-matched, so `%k:f` should work on that branch
